BCI_analysis/plot/plot_imaging.py

```diff
@@ -263,3 +264,95 @@ def plot_trialwise_closed_loop(dataDict, dayOfInterest):
     plt.show()
 
 
+def correlation_deltaActivity(dataDict, dayOfInterest):
+    ## Set Up Day of Session
+    # dayOfInterest = input('Day of interest (day 1 = -1... dont do day 1): ')
+    if int(dayOfInterest) == 0:
+        while int(dayOfInterest) <= 0: #this is so we dont get stupid values
+            print('Incorrect Day Entry, redo please')
+            dayOfInterest = input('Day of interest (Day 1 = "1") ')
+    day = int(dayOfInterest) - 1 #Since python starts at 0, we are subtracting one to account for indexing
```

Comment:
Its best we assume that the value passed is already a python indexed value. I have also assumed this, so has Marton, will help keep everything consistent